src/Test.Utilities/CSharpCodeFixVerifier`2.cs

```diff
@@ -23,11 +23,15 @@ public static DiagnosticResult Diagnostic(string diagnosticId)
         public static DiagnosticResult Diagnostic(DiagnosticDescriptor descriptor)
             => CSharpCodeFixVerifier<TAnalyzer, TCodeFix, XUnitVerifier>.Diagnostic(descriptor);
 
-        public static async Task VerifyAnalyzerAsync(string source, params DiagnosticResult[] expected)
+        public static Task VerifyAnalyzerAsync(string source, params DiagnosticResult[] expected)
```

Comment:
⚠️  This needs to be marked `async`, or calling code will not report a warning if the `await` is missing

Reply:
Not changed by this PR but do you want me to update the `VerifyCodeFixAync`? They have the same "issue".

Reply:
Sure, that one was my mistake!